server/src/sim/makeworkbook.py

```diff
@@ -628,3 +628,34 @@ def create(self, path):
                 self.formats.write_unlocked(self.sheet, row+2, i, data)
             row+=1
         self.book.close()
+
```

Comment:
Can't we refactor this and the other GraphTable into one class? The GraphTable with one page looks like a specific case of OptimaMultiSheetGraphTable. That would reduce the amount of duplicated code.